data_changes.md

```diff
@@ -0,0 +1,58 @@
+## Changes to Imported Data ##
+
+Generally, data will be imported as-is from historical data or newly given data
+with a few exceptions:
+
+- _IMPORT TYPE_: Broker, USASpending
+- _CHANGE_: **Creation of summary awards**
```

Comment:
If you have any ideas, feel free. I'm not sure how to word this stuff in a way that someone unfamiliar with generally how the data is to understand it